packages/ckeditor5-build-classic/src/ckeditor.js

```diff
@@ -37,11 +37,17 @@ import TextTransformation from '@ckeditor/ckeditor5-typing/src/texttransformatio
 import CloudServices from '@ckeditor/ckeditor5-cloud-services/src/cloudservices';
 import Style from '@ckeditor/ckeditor5-style/src/style';
 import SpecialCharacters from '@ckeditor/ckeditor5-special-characters/src/specialcharacters';
-import SpecialCharactersEssentials from '@ckeditor/ckeditor5-special-characters/src/specialcharactersessentials';
+import SpecialCharactersCurrency from './specialcharacterscurrency';
+import SpecialCharactersLatin from './specialcharacterslatin';
+import SpecialCharactersMathematical from './specialcharactersmathematical';
+import SpecialCharactersText from './specialcharacterstext';
 
 class ClassicEditor extends ClassicEditorBase {}
 class InlineEditor extends InlineEditorBase {}
 
+const SpecialCharactersEssentials = [
```

Comment:
@hilhorstt is this what you mean? could this be a solution to the issue. I have removed the import of specialCharactersEssentials and created and array with special characters without the arrays

Reply:
Yes, I would say this might work. Then you can add a list of characters in grading-frontend, instead of in `packages/ckeditor5-special-characters/src/specialcharactersarrows.js`, otherwise you'd have to make a custom build for that plugin.